projects/storefrontlib/cms-components/product/product-intro/product-intro.component.ts

```diff
@@ -72,12 +72,13 @@ export class ProductIntroComponent {
         const tabsComponent = this.getTabsComponent();
         const reviewsTab =
           tabsComponent && this.getTabByLabel(reviewsTabLabel, tabsComponent);
-
         if (reviewsTab) {
-          this.clickTabIfInactive(reviewsTab);
           setTimeout(() => {
-            reviewsTab.scrollIntoView({ behavior: 'smooth' });
-            reviewsTab.focus({ preventScroll: true });
+            this.clickTabIfInactive(reviewsTab);
```

Comment:
Changes to the existing functionalities are considered breaking. You can see examples and how to deal with them here: https://wiki.one.int.sap/wiki/display/spar/Spartacus+Onboarding in the `Backwards Incompatibility` section. You can also look at other implementations of feature flags in the project e.g: `ScrollToTopComponent`

Reply:
I've added a feature flag: a11yScrollToReviewByShowReview.